util/unifyfs/src/unifyfs-rm.c

```diff
@@ -807,11 +869,25 @@ int unifyfs_start_servers(unifyfs_resource_t* resource,
         return rc;
     }
 
-    if (args->script != NULL) {
-        return script_launch(resource, args);
-    } else {
-        return resource_managers[resource->rm].launch(resource, args);
+    pid = fork();
+    if (pid < 0) {
+        fprintf(stderr, "failed to launch the server process (%s)\n",
```

Comment:
error message isn't quite accurate. how about "failed to create the server launch process"?